app/components/publishers/vacancies_component.rb

```diff
@@ -18,8 +18,24 @@ def render?
     @organisation.all_vacancies.active.any?
   end
 
-  def selected_class(vacancy_type)
-    "govuk-tabs__list-item--selected" if @selected_type == vacancy_type
+  def selected(vacancy_type)
+    @selected_type == vacancy_type
+  end
+
+  def sortable
```

Comment:
`sortable?`

Reply:
it is really is_sortable but i got picked up on this before that it is not the ruby convention to prepend with is_ ???

Reply:
oh no sorry i see your point, its boolean and needs ?